fabtests/common/shared.c

```diff
@@ -374,8 +374,9 @@ static int ft_reg_mr(void *buf, size_t size, uint64_t access,
 	      !(opts.options & FT_OPT_USE_DEVICE)) ||
 	     (!(fi->domain_attr->mr_mode & FI_MR_HMEM) &&
 	      opts.options & FT_OPT_USE_DEVICE)) &&
-	    !(fi->caps & (FI_RMA | FI_ATOMIC)))
```

Comment:
This looks like an unnecessary change

Reply:
Addressed in latest changes. I had put a print statement here so I had to put parentheses and then forgot to remove the parentheses.